srml/support/src/event.rs

```diff
@@ -227,6 +227,9 @@ macro_rules! __decl_generic_event {
 		{ $( $events:tt )* };
 		{ ,$( $generic_param:ident = $generic_type:ty ),* };
 	) => {
+		/// Accessor to [`RawEvent`]
```

Comment:
Hmm, I'm not quite happy with this explanation.

Reply:
`RawEvent` specialized for the configuration `Trait`

Reply:
I have written as such